src/main/authentication/adminAuthentication.ts

```diff
@@ -13,4 +18,19 @@ export class AdminAuthentication {
     }
     return false;
   }
+
+  public forgotPasswordRedirect(req, res, next): void {
+    const body = JSON.stringify(req.body);
+    if (body.includes('AADB2C90118')) {
+      const CLIENT_ID = config.get('secrets.pip-ss-kv.CLIENT_ID');
```

Comment:
Hey Danny, I think it looks a bit easier to read if all the URL query parameters are laid out in json. It is not crucial to change but just a suggestion. It is up to u :) 

const params = new URLSearchParams({
  ‘p’: `${authenticationConfig.FORGOT_PASSWORD_POLICY}`,
  'client_id': config.get('secrets.pip-ss-kv.CLIENT_ID'),
  ‘nonce’: ‘defaultNonce’,
  'redirect_uri': `${FRONTEND_URL}/password-change-confirmation`,
  'scope': ‘openid’,
  ‘responseType’:’id_token’,
  ‘prompt’: 'login'
});
res.redirect(`${B2C_URL}/oauth2/v2.0/authorize? ` + params.toString());

Reply:
Hey Kian - I 100% agree with you on this, it's a much nicer solution. However, I'm concerned about the limited time we have, so I will add it in my next (less stressfully timed) frontend PR - good compromise?

Reply:
Sure thing. Will resolve the comment :)